tests/benchmark/bloatnet/deploy_create2_factory.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+"""
+Deploy a simple CREATE2 factory for benchmark tests.
+This factory can be reused across all tests and allows deterministic addresses.
+"""
+
+import argparse
+
+from eth_utils import keccak
+from web3 import Web3
+
+
+def deploy_factory(rpc_url: str):
+    """Deploy a minimal CREATE2 factory."""
+    # Connect to Geth
+    w3 = Web3(Web3.HTTPProvider(rpc_url))
+    if not w3.is_connected():
+        print(f"Failed to connect to {rpc_url}")
+        return None
+
+    test_account = w3.eth.accounts[0]
+    print(f"Using test account: {test_account}")
+
+    # Minimal CREATE2 factory bytecode
+    # Takes salt (first 32 bytes) and bytecode (rest) from calldata
+    # Returns the deployed address
+    factory_bytecode = bytes(
+        [
+            # Runtime code
```

Comment:
I strongly suggest to read a bit about the tooling EEST provides to write these kind of bytecode tests. 

For instance, here is CREATE2 factory code: https://github.com/ethereum/execution-spec-tests/blob/291fe00e65d82e05ca2bd5e7a49e4ac4a8c7a5b9/tests/benchmark/test_worst_bytecode.py#L142-L160 which will factory-deploy this initcode: https://github.com/ethereum/execution-spec-tests/blob/291fe00e65d82e05ca2bd5e7a49e4ac4a8c7a5b9/tests/benchmark/test_worst_bytecode.py#L118-L137 which itself gets called from another contract which will invoke the factory contract N times (calldata input) https://github.com/ethereum/execution-spec-tests/blob/291fe00e65d82e05ca2bd5e7a49e4ac4a8c7a5b9/tests/benchmark/test_worst_bytecode.py#L165-L168.

Reply:
EEST will else help with things like formatting numbers for you, i.e. 0xff gets put in a PUSH1 but 0xffff will be put in a PUSH2 (so you dont have to do the bitwise logic like the shifting and the AND masks here)

Reply:
To be clear, If STEEL team doesn't want to have the deployment helper files within EEST (which I could def understand), then I'd prefer avoiding the dependency on EEST for the standalone files. (As I'll publish them as a gist or similar so that they can be executed after a wget or whatever).

If they remain here, I agree the refactor makes sense. Pushed it for you all to see the diff in: cf2c7c6